WordPress/Classes/ViewRelated/Post/AuthorFilterButton.swift

```diff
@@ -34,7 +34,8 @@ final class AuthorFilterButton: UIControl {
         didSet {
             switch filterType {
             case .everyone:
-                authorImageView.image = UIImage(named: "icon-people")
+                authorImageView.image = UIImage(named: "icon-people")?.withRenderingMode(.alwaysTemplate)
+                authorImageView.tintColor = .text
```

Comment:
**Optional**: I think this could be a one-liner with the `withTintColor(_ color: UIColor, renderingMode: UIImage.RenderingMode)` function:

```suggestion
                authorImageView.image = UIImage(named: "icon-people")?.withTintColor(.text, renderingMode: .alwaysTemplate)
```